### PR TITLE
udpate #245 英文對齊 開發型容器 4

### PR DESCRIPTION
**Pull request by chuchun211**

### CODE REVIEW

File: i18n/en/docusaurus-plugin-content-docs/current/ccs_framwork_image/matlab-byol.md [hunk 1]
```diff
@@ -10,14 +10,13 @@ sync_original_preview: 'https://man.twcc.ai/@preview-twccdocs/ccs-concept-image-
 # <img style={{width:55+'px'}} src='https://cos.twcc.ai/SYS-MANUAL/uploads/upload_3022d6e6790c870e499eac8e2c77d53c.png' /> Matlab (BYOL)
 
 
-
-TWCC provides Matlab in preview now. To run Matlab on the cloud, you need to enter your license - MathWorks account and password.
+TWCC now offers a preview version of Matlab. To run Matlab on the cloud, you need to enter your license, i.e. your MathWorks account and password.
```
Comment (MiaYangTWS) [inline i18n/en/docusaurus-plugin-content-docs/current/ccs_framwork_image/matlab-byol.md:13]: Matlab已經正式提供、不是preview版囉，TWCC平台中，CCS映像檔敘述的內容如下:

TWCC provides pay-as-you-go working environment of [NGC optimized MATLAB](https://man.twcc.ai/@twccdocs/ccs-concept-image-main-en/%2F%40twccdocs%2Fccs-concept-image-matlab-en) (preview). MATLAB® is a mathmatical computing software that allows matrix manipulations, plotting of functions and data. Moreover, you can interface MATLAB with other languages. To activate the service on the cloud, please enter your MathWorks account and password associated with a valid license.

image

**Review by MiaYangTWS — COMMENTED** [review]
謝謝協助校閱英文語意更順暢了，有一項與實際有較大差異的內容，請修改內容，謝謝。

File: i18n/en/docusaurus-plugin-content-docs/current/ccs_framwork_image/matlab-byol.md (hunk 1 above)
Comment (MiaYangTWS) [reply i18n/en/docusaurus-plugin-content-docs/current/ccs_framwork_image/matlab-byol.md]: 不好意思，沒注意到內容還有"(preview)" 請移除

Comment (chuchun211) [reply i18n/en/docusaurus-plugin-content-docs/current/ccs_framwork_image/matlab-byol.md]: > Matlab已經正式提供、不是preview版囉，TWCC平台中，CCS映像檔敘述的內容如下:
> 
> TWCC provides pay-as-you-go working environment of [NGC optimized MATLAB](https://man.twcc.ai/@twccdocs/ccs-concept-image-main-en/%2F%40twccdocs%2Fccs-concept-image-matlab-en) (preview). MATLAB® is a mathmatical computing software that allows matrix manipulations, plotting of functions and data. Moreover, you can interface MATLAB with other languages. To activate the service on the cloud, please enter your MathWorks account and password associated with a valid license.
> 
> image

已更新！補了中文文件的部分 #267

**Review by MiaYangTWS — APPROVED** [review]

**Pull request merged**